components/Nav.js

```diff
@@ -1,120 +1,176 @@
-import 'bootstrap/dist/css/bootstrap.min.css'
-
+import * as React from 'react'
+import AppBar from '@mui/material/AppBar'
+import Box from '@mui/material/Box'
+import Toolbar from '@mui/material/Toolbar'
+import IconButton from '@mui/material/IconButton'
+import Typography from '@mui/material/Typography'
+import Menu from '@mui/material/Menu'
+import MenuIcon from '@mui/icons-material/Menu'
+import Container from '@mui/material/Container'
+import Button from '@mui/material/Button'
+import MenuItem from '@mui/material/MenuItem'
+import { makeStyles } from '@mui/styles'
 import Image from 'next/image'
 import Link from 'next/link'
-import { useState } from 'react'
-import Button from 'react-bootstrap/Button'
-
 import orcasoundlogo from '../images/orcasoundlogo.png'
-import navStyles from '../styles/Nav.module.css'
+import NotificationsIcon from '@mui/icons-material/Notifications'
+
+const navLinks = [
+  {
+    name: 'About Us',
+    url: '/about',
+    icon: '',
+  },
+  {
+    name: 'Listen',
+    url: '/listen',
+    icon: '',
+  },
+  {
+    name: 'Learn',
+    url: '/learn',
+    icon: '',
+  },
+  {
+    name: 'Get Involved',
+    url: '/getinvolved',
+    icon: '',
+  },
+]
+
+const useStyles = makeStyles({
+  nav: {
+    backgroundColor: '#080d26',
+  },
+  navLogoUrl: {
+    width: '60px',
+    height: '44px',
+    top: '20px',
+    left: '15px',
+    borderRadius: '100px',
+    position: 'absolute',
+  },
+  logo: { flexGrow: 1 },
+  logoContainer: {
+    backgroundColor: 'white',
+    width: '69px',
+    height: '66px',
+    margin: '10px',
+    borderRadius: '100px',
+  },
+  link: {
+    color: 'white',
+    '&:hover': {
+      textDecoration: 'none',
+      color: 'white',
+    },
+  },
+})
+
+const Nav = () => {
+  const [anchorElNav, setAnchorElNav] = React.useState(null)
```

Comment:
This works fine but it might be simpler just to store open state as boolean and pass the anchor ref in directly (since it's always the same icon)